api/src/pcapi/core/educational/repository.py

```diff
@@ -109,6 +109,35 @@ def get_and_lock_educational_deposit(
     return educational_deposit
 
 
+def get_ministry_budget_for_year(ministry: str | None, educational_year_id: str) -> Decimal:
```

Comment:
Pourquoi on gère le cas où c'est null ? Car on a pas de contrainte en db ?

Reply:
pour faire plaisir a mypy. En pratique ça ne change rien. Si on passe None ça retourne Decimal(0)

Reply:
apres verification, en effet la colonne est nullable. ça pourrait être une bonne chose de la rendre NOT NULL mais pas dans ce ticket